Objects/Objects/BuiltElements/Topography.cs

```diff
@@ -46,3 +47,10 @@ public RevitTopography([SchemaMainParam] Mesh displayMesh, List<Parameter> param
     public Base parameters { get; set; }
   }
 }
+
+namespace Objects.GIS
+{
+  public class GisTopography : Topography
+  {
+  }
+}
```

Comment:
I'm aware some other files have mixed namespaces... I think what you had before (Objects/GIS/GisTopography) was great so no need to undo that, just change it's inheritance from `Base` to `Topography` like here.

We'd also want the typed properties in the class (band_count, band_names... etc) even if we're not currently using them for anything in revit. They should still exist ;)